tests/suites/static_analysis/lint_go.sh

```diff
@@ -38,6 +38,20 @@ run_domain_imports() {
 	done
 }
 
+run_juju_errors_imports() {
+    pkgs=("domain")
+
+    for pkg in "${pkgs[@]}"; do
+        dirs=$(find ${pkg} -mindepth 1 -maxdepth 10 -type d | sort -u)
+        for subdir in $dirs; do
+            echo "$subdir"
```

Comment:
nit: would love to stay consistent
`run_domain_imports()` does this
```
	for dir in $dirs; do
		if [[ ! -d $dir ]]; then
			continue
		fi
		echo "Checking $dir"
```

Could we rename `subdir` to `dir` and use `echo "Checking $dir"`

A drive by to remove the `if [[ ! -d $dir ]]; then` check would be great since `run_domain_imports()` does do `find -type d` in the first place

Reply:
Done & Done